openquake/hazardlib/gsim/pankow_pechmann_2004.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2014-2017 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`PankowPechmann2004`.
+"""
+from __future__ import division
+
+import numpy as np
+from scipy.constants import g as gravity
+
+from openquake.hazardlib.gsim.base import GMPE, CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import PGA, PGV, SA
+
+
+class PankowPechmann2004(GMPE):
+    """
+    Implements GMPE developed by Kris L. Pankow and James C. Pechmann 
+    and published as "The SEA99 Ground-Motion Predictive Relations for Extensional
+    Tectonic Regimes: Revisions and a New Peak Ground Velocity Relation"
+    Bulletin of the Seismological Society of America,
+    Vol. 94, No. 1, pp. 341–348, February 2004
+    """
```

Comment:
Here you can insert a line `non_verified = True`. Then the user of this GSIM will get a warning
```
PankowPechmann2004 is not independently verified - the user is liable for their application
```
which probably is what you want. We have example of several non_verified GSIMs in hazardlib.